lib/index.ts

```diff
@@ -1,30 +1,25 @@
-import { JellyfishPluginBase } from '@balena/jellyfish-plugin-base';
-import { actions } from './actions';
-import { cards } from './cards';
-import integrations from './integrations';
+import type { PluginDefinition } from '@balena/jellyfish-worker';
+import { contracts } from './contracts';
+import { integrations } from './integrations';
+
+// tslint:disable-next-line: no-var-requires
+const { version } = require('../package.json');
 
 /**
  * The Front Jellyfish plugin.
  */
-export class FrontPlugin extends JellyfishPluginBase {
-	constructor() {
-		super({
-			slug: 'jellyfish-plugin-front',
-			name: 'Front Plugin',
-			version: '1.0.0',
-			actions,
-			cards,
-			integrations,
-			requires: [
-				{
-					slug: 'action-library',
-					version: '>=15.x',
-				},
-				{
-					slug: 'jellyfish-plugin-default',
-					version: '>=19.x',
-				},
-			],
-		});
-	}
-}
+export const frontPlugin = (): PluginDefinition => {
```

Comment:
This can be a constant

Reply:
Sorry, not sure what you mean.

Reply:
I guess it doesn't matter since the other plugins are doing the same thing, but `frontPlugin` doesn't need to be a function. It's just a constant.